docs/iris/src/whatsnew/latest.rst

```diff
@@ -131,8 +137,8 @@ Incompatible Changes
   duplicate coordinate errors in certain circumstances. (:pull:`3718`)
 
 
-Deprecations
-============
+🚫 Deprecations
```

Comment:
```suggestion
🗑️ Deprecations
```
or
```suggestion
🔥 Deprecations
```
... something that's more distinct from the ⛔ of `Incompatible Changes`

Reply:
Given the choice, I'd go with some flamage. Great suggestion :+1:

Reply:
@trexfeathers ... now you've started something :wink: 

Going on the flame theme, I've changed `Incompatible Changes` to 💣 (bomb) ... almost suggested 💥 (collision), but again that's too similar to 🔥 `Deprecations`